cutscene_engine/cutscene_scenes/cutscene_test2.tscn

```diff
@@ -0,0 +1,238 @@
+[gd_scene load_steps=14 format=2]
```

Comment:
Test file

Reply:
this one serves the same purposes as the first one. Can be scrapped